src/Controller/AboutController.php

```diff
@@ -310,9 +310,11 @@ public function peopleAction(Request $request, string $type) : Response
                 $arguments['lists'][] = $this->createAboutProfiles($people->forType('reviewing-editor'), 'Reviewing editors', true);
         }
 
-        $arguments['contentHeader'] = all(['types' => $types, 'title' => promise_for($arguments['title'])])
+        $arguments['contentHeader'] = all(['types' => $types, 'title' => promise_for($arguments['title']), 'subject' => promise_for($arguments['subject'] ?? null)])
             ->then(function (array $parts) use ($arguments) {
-                return new ContentHeader($parts['title'], null, 'The working scientists who serve as eLife editors, our early-career advisors, governing board, and our executive staff all work in concert to realise eLife’s mission to accelerate discovery',
+                $aimsScope = ($parts['subject'] && $parts['subject']->getAimsAndScope()->offsetExists(0)) ? $parts['subject']->getAimsAndScope()->offsetGet(0)->getText() : null;
+
+                return new ContentHeader($parts['title'], null, $aimsScope ?? 'The working scientists who serve as eLife editors, our early-career advisors, governing board, and our executive staff all work in concert to realise eLife’s mission to accelerate discovery',
```

Comment:
`s/accelerate discovery',/accelerate discovery.',`: default text should terminate with a full stop to be consistent with the other text that gets inserted here now.